src/pageObject.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
-class PageObject {
+class PageObject extends Header {
```

Comment:
`Page Object` shouldn't extend `Header`, there was no such requirement in task description